src/test/regress/input/multi_alter_table_statements.source

```diff
@@ -162,7 +162,54 @@ FROM
 ORDER BY attnum;
 \c - - - :master_port
 
+-- check we don't intercept DDL commands if propagation is turned off
```

Comment:
> actually we do not start with capital letters all the time. almost all single line comments in test files start in small caps.

Interesting. We start with capital letters in this file, and that was my impression overall. But, you're right in general.